abstract/test.js

```diff
@@ -1,23 +1,20 @@
 'use strict'
 
-var path = require('path')
-var location = path.join(__dirname, 'level-test-' + process.pid + '.db')
-
 module.exports = function (test, level, options) {
   options = options || {}
 
-  require('./base-test')(test, level, location)
-  require('./db-values-test')(test, level, options.nonPersistent, location)
+  require('./base-test')(test, level)
```

Comment:
Memoizing `location` means dependents can do e.g.:

```
require('level-packager/abstract/base-test')(test, level)
require('level-packager/abstract/db-values-test')(test, level)
// etc
```

Reply:
But it also means they can't customize it if they need to?

Reply:
That's correct, but they never could.

Reply:
We can add `location` parameter, which defaults to the current location.

Reply:
> That's correct, but they never could.

OK, let's leave it be then

Reply:
It should be easy to fix later on if needed.